.gitignore

```diff
@@ -98,3 +98,4 @@ public/system/
 
 ## Sitemap
 public/sitemap.xml
+.byebug_history
```

Comment:
Since the project requires byebug, I think it's okay to put .byebug_history in the .gitignore.